com.reprezen.swagedit/src/com/reprezen/swagedit/schema/SwaggerSchema.java

```diff
@@ -171,13 +173,15 @@ public void allowJsonRefInContext(String jsonReferenceContext, boolean allow) {
         if (definition == null) {
             throw new IllegalArgumentException("Invalid JSON Reference Context: " + jsonReferenceContext);
         }
-        JsonNode refNode = mapper.createObjectNode().put("$ref", "#/definitions/elementRef");
+        int lastIndex = definition.size() - 1;
```

Comment:
I'm not crazy about the assumption that this will always be the last oneOf element. I know that in current code it will be, but perhaps it's an assumption that will be unwittingly violated in some future code, leading to sporadic failures. Might be better to do a List#contains test on the entire oneOf array. Slightly slower, but this only happens when preferences change.